src/Raven.Server/Storage/Schema/Updates/Server/From11.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections.Generic;
+using Sparrow.Json;
+using Voron;
+using Voron.Data.Tables;
+using static Raven.Server.ServerWide.ClusterStateMachine;
+using static Raven.Server.Documents.DocumentsStorage;
+using static Raven.Server.ServerWide.Commands.CompareExchangeCommandBase;
+
+namespace Raven.Server.Storage.Schema.Updates.Server
+{
+    public unsafe class From11 : ISchemaUpdate
+    {
+        public bool Update(UpdateStep step)
+        {
+            var dbs = new List<string>();
+            const string dbKey = "db/";
+
+            var oldCompareExchangeSchema = new TableSchema().
+                DefineKey(new TableSchema.SchemaIndexDef
+            {
+                StartIndex = (int)UniqueItems.Key,
+                Count = 1
+            });
+
+            var items = step.ReadTx.OpenTable(ItemsSchema, Items);
+            using (Slice.From(step.ReadTx.Allocator, dbKey, out Slice loweredPrefix))
+            {
+                foreach (var result in items.SeekByPrimaryKeyPrefix(loweredPrefix, Slices.Empty, 0))
```

Comment:
You aren't limiting this to just `db/` prefixes here

Reply:
loweredPrefix slice is made from "db/" string.

Reply:
And after you run through the `db/` stuff, you'll read other values.
The prefix you pass it empty here.